examples/computer_vision/efficientdet_pytorch/model_def.py

```diff
@@ -0,0 +1,394 @@
+            distributed=False,
+            pin_mem=False,
+            anchor_labeler=None,
+        ):
+
+            if isinstance(input_size, tuple):
+                img_size = input_size[-2:]
+            else:
+                img_size = input_size
+
+            if is_training:
+                transform = transforms_coco_train(
+                    img_size,
+                    interpolation=interpolation,
+                    use_prefetcher=use_prefetcher,
+                    fill_color=fill_color,
+                    mean=mean,
+                    std=std
+                )
+            else:
+                transform = transforms_coco_eval(
+                    img_size,
+                    interpolation=interpolation,
+                    use_prefetcher=use_prefetcher,
+                    fill_color=fill_color,
+                    mean=mean,
+                    std=std
+                )
+
+            dataset.transform = transform
+
+            sampler = None
+
+            collate_fn = DetectionFastCollate(anchor_labeler=anchor_labeler)
+            loader = DataLoader(
+                dataset,
+                batch_size=self.context.get_per_slot_batch_size(),
+                shuffle=False, #sampler is None and is_training,
+                num_workers=num_workers,
+                sampler=sampler,
+                pin_memory=pin_mem,
+                collate_fn=collate_fn,
+            )
+
+            return loader
+
+    def build_training_data_loader(self):
+
+        dataset_train, self.dataset_eval = create_dataset(self.args.dataset, self.args.root)
+
+        self.labeler = None
+        if not self.args.bench_labeler:
+            self.labeler = AnchorLabeler(Anchors.from_config(self.model_config), self.model_config.num_classes, match_threshold=0.5)
+
+        loader_train = self._create_loader(
+            dataset_train,
+            input_size=self.input_config['input_size'],
+            batch_size=self.context.get_per_slot_batch_size(),
+            is_training=True,
+            use_prefetcher=self.args.prefetcher,
+            re_prob=self.args.reprob,
+            re_mode=self.args.remode,
+            re_count=self.args.recount,
+            # color_jitter=self.args.color_jitter,
+            # auto_augment=self.args.aa,
+            interpolation=self.args.train_interpolation or self.input_config['interpolation'],
+            fill_color=self.input_config['fill_color'],
+            mean=self.input_config['mean'],
+            std=self.input_config['std'],
+            num_workers=self.args.workers,
+            distributed=self.args.distributed,
+            pin_mem=self.args.pin_mem,
+            anchor_labeler=self.labeler,
+        )
+        if self.model_config.num_classes < loader_train.dataset.parser.max_label:
+            logging.error(
+                f'Model {self.model_config.num_classes} has fewer classes than dataset {loader_train.dataset.parser.max_label}.')
```

Comment:
From a style perspective, might be good to have the last parenthesis on a different line. Ditto below.

Reply:
done